src/Dapr.Client/DaprClient.cs

```diff
@@ -783,6 +783,75 @@ public async Task<StateEntry<TValue>> GetStateEntryAsync<TValue>(string storeNam
             StateOptions stateOptions = default,
             IReadOnlyDictionary<string, string> metadata = default,
             CancellationToken cancellationToken = default);
+        /// <summary>
+        /// Define a SaveStateAsync task that gets a binary input   
+        /// </summary>
+        /// <param name="storeName">The  </param>
+        /// <param name="key"></param>
+        /// <param name="binaryValue"></param>
+        /// <param name="stateOptions"></param>
+        /// <param name="metadata"></param>
+        /// <param name="cancellationToken"></param>
+        /// <returns></returns>
+        public abstract Task SaveStateByteAsync(
```

Comment:
Just wondering whether these new save methods should be overloads of `SaveStateAsync()`?  (I see we already have a `PublishByteEventAsync()` which sets a precedent for a new method name, though.  Given that, I'd suggest `SaveByteStateAsync()` or `SaveBinaryStateAsync()`.  Honestly, though, I feel like `PublishByteEventAsync()` could have been an overload of `PublishEventAsync()` as well.)

Reply:
Thanks for your comments. I have renamed the methods similar to SaveByteStateAsync.   Usability concern if we add this as overloaded methods to 'SaveStateAsync<Tvalue>'. The only input changing here is Tvalue to ReadOnlyMemory<byte>. Unless we explicitly declare the input variable as ReadOnlyMemory<byte>, error is thrown as the system tries to use SaveStateAsync<Tvalue>.